pkg/model/dev.go

```diff
@@ -299,6 +296,35 @@ func Read(bytes []byte) (*Dev, error) {
 	return dev, nil
 }
 
+func loadAbsPath(folder, path string) string {
+	if filepath.IsAbs(path) {
+		return path
+	}
+	return filepath.Join(folder, path)
+}
+
+func (dev *Dev) loadAbsPaths(devPath string) error {
+
+	devDir, err := filepath.Abs(filepath.Dir(devPath))
+	if err != nil {
+		return err
+	}
+
+	dev.Image.Context = loadAbsPath(devDir, dev.Image.Context)
+	dev.Image.Dockerfile = loadAbsPath(devDir, dev.Image.Dockerfile)
+	dev.Push.Context = loadAbsPath(devDir, dev.Push.Context)
+	dev.Push.Dockerfile = loadAbsPath(devDir, dev.Push.Dockerfile)
+
+	for i := range dev.Volumes {
+		if dev.Volumes[i].LocalPath == "" {
+			continue
+		}
+		dev.Volumes[i].LocalPath = loadAbsPath(devDir, dev.Volumes[i].LocalPath)
```

Comment:
are we going to support absolute paths on the left? (e.g. to sync a folder that it's not on the repo where I'm running `okteto up` from, like $HOME/.ssh).